embassy-stm32/Cargo.toml

```diff
@@ -80,6 +80,7 @@ defmt = ["dep:defmt", "bxcan/unstable-defmt", "embassy-sync/defmt", "embassy-exe
 memory-x = ["stm32-metapac/memory-x"]
 subghz = []
 exti = []
+device-signature = []
```

Comment:
Is a feature necessary? This does not introduce any fat dependencies and is quite small.